components/settings/settingsAccount.tsx

```diff
@@ -214,139 +216,230 @@ export default function SettingsAccount({ user, userConfig }: SettingsAccountPro
     }
   }
 
+  // NotificationType is an enum
+  const allNotifs = Object.values(NotificationType);
+
+  const notifLabels = {
```

Comment:
Can we create a new tab `/settings/notifications` for all of these + the daily digest? I think it'll be a bit cleaner

Reply:
sure